### PR TITLE
Add ipv6 only mgmt syslog test case

### PR DESCRIPTION
**Pull request by maipbui**
### Description of PR

Summary: Add ipv6 only mgmt syslog test case
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311

#### What is the motivation for this PR?
Add ipv6 only mgmt syslog test case
#### How did you verify/test it?
Run tests

### CODE REVIEW

File: tests/ip/test_mgmt_ipv6_only.py [hunk 1]
```diff
@@ -78,3 +87,75 @@ def test_image_download_ipv6_only(creds, duthosts, enum_dut_hostname,
...
+        logger.debug("Added new rsyslog server IP {}".format(dummy_syslog_server_ip_a))
+    if dummy_syslog_server_ip_b is not None:
+        if "201911" in duthost.os_version and ":" in dummy_syslog_server_ip_b:
+            pytest.skip("IPv6 syslog server IP not supported on 201911")
+        duthost.shell("sudo config syslog add {}".format(dummy_syslog_server_ip_b))
+        logger.debug("Added new rsyslog server IP {}".format(dummy_syslog_server_ip_b))
+
+    logger.info("Start tcpdump")
+    # Make sure that the DUT_PCAP_FILEPATH dose not exist
+    duthost.shell("sudo rm -f {}".format(DUT_PCAP_FILEPATH))
```
Comment (qiluo-msft) [inline tests/ip/test_mgmt_ipv6_only.py:125]: >shell [](http://example.com/codeflow?start=12&length=5)

Is there a safer version of this function?

File: tests/ip/test_mgmt_ipv6_only.py [hunk 2]
```diff
@@ -78,3 +87,75 @@ def test_image_download_ipv6_only(creds, duthosts, enum_dut_hostname,
     else:
         pytest.fail("Failed to download image from image_url {} via any of {}"
                     .format(image_url, list(mgmt_interfaces)))
+
+
+@pytest.mark.parametrize("dummy_syslog_server_ip_a, dummy_syslog_server_ip_b",
+                         [("fd82:b34f:cc99::100", None),
+                          ("fd82:b34f:cc99::100", "fd82:b34f:cc99::200")])
+def test_syslog(rand_selected_dut, dummy_syslog_server_ip_a, dummy_syslog_server_ip_b,
```
Comment (qiluo-msft) [inline tests/ip/test_mgmt_ipv6_only.py:95]: >test_syslog [](http://example.com/codeflow?start=4&length=11)

Please reuse tests/syslog/test_syslog.py::tests/syslog/test_syslog.py

**Review by qiluo-msft — APPROVED** [review]
we can still improve the fixture, and not add a new testcase or test file. Just modify the fixture parameters for exsiting testcases.

**Review by wen587 — APPROVED** [review]
LGTM

**Comment by maipbui** [comment]
@StormLiangMS could you help cherry-pick this PR? thanks!

**Pull request merged**